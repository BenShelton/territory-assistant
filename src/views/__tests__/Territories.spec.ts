import { mount } from '@vue/test-utils'

import Territories from '../Territories.vue'

describe('Views/Territories', () => {
  it('should have the name "Territories"', () => {
    const wrapper = mount(Territories)
    expect(wrapper.name()).toBe('Territories')
  })

  it('should match the snapshot', () => {
    const wrapper = mount(Territories)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
