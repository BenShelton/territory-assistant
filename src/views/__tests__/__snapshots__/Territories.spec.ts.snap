// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Views/Territories should match the snapshot 1`] = `
<div class="container">
  <h1 class="headline">
    Territories
  </h1>
</div>
`;
